dotcom-rendering/src/web/components/MiniCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from '@emotion/react';
+import { body } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { DCRContainerPalette } from 'src/types/front';
+import type { TrailType } from 'src/types/trails';
+import { generateImageURL } from '../../lib/generateImageURL';
+import { ContainerOverrides } from '../../types/palette';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+
+const imageStyles = css`
+	width: 120px;
+	padding-right: 10px;
+`;
+
+const linkStyles = css`
+	${body.medium()}
+	font-weight: bold;
+	display: flex;
+	align-items: flex-start;
+`;
+
+const linkOverrideStyles = (containerOverrides?: ContainerOverrides) => css`
+	${containerOverrides && `color: ${containerOverrides.text.cardHeadline}`}
+`;
+
+type Props = {
+	trail: TrailType;
+	showImage: boolean;
+	containerPalette?: DCRContainerPalette;
+};
+
+/**
+ * A simpler form of card that does not have any of the complexities of a normal card such as sublinks, kickers, format, etc.
+ *
+ * Primarily used for rendering Snaps in containers such as @see NavList
```

Comment:
I think `@linkcode` will work better

```suggestion
 * Primarily used for rendering Snaps in containers such as {@linkcode NavList}
```